corehq/apps/app_manager/static/app_manager/js/app_manager.js

```diff
@@ -279,83 +265,106 @@ hqDefine('app_manager/js/app_manager', function () {
             }
             $sortable.sortable(options);
         }
-
         function updateAfterMove(e, ui, $sortable) {
             // because the event is triggered on both sortables when moving between one sortable list to
             // another, do a check to see if this is the sortable list we're moving the item to
-            if ($sortable.find(ui.item).length < 1) {
-                return;
-            }
+            if ($sortable.find(ui.item).length < 1) { return; }
```

Comment:
I abhor this style...maybe we can argue about it in book club.

Reply:
which style?  doing this on one line, or short-circuiting early?

Reply:
the single line